### DISCUSSION TITLE
[table] Disabled row doesn't propertly work in case of definition table

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1934] [Collections (form, grid, menu, message, table, ...)] [answered]
`disabled` class for row in a table doesn't propertly work in case of use of `definition` class.
It has no effect on the first `td`, that in case of definition table, has bolder text.

![image](https://user-images.githubusercontent.com/5399265/108356188-1bf20b00-71ec-11eb-981a-8668ab43890d.png)

https://jsfiddle.net/4syvoqnt

**Comment by lubber-de** [comment, marked as answer]
There exists an undocumented feature: add the class `ignored` to the first column where the row is disabled, like this

```html
<tr class="disabled">
      <td class="ignored">reset rating</td>
</tr>
```
See https://jsfiddle.net/lubber/ve02mwad/2/
![image](https://user-images.githubusercontent.com/18379884/108357410-b141cf00-71ed-11eb-8651-53e9d6b5b0e6.png)

**Comment by smartm0use** [comment]
Great, thank you!
